inc/class-clean-emails.php

```diff
@@ -86,6 +87,11 @@ public function comment_notification_text( $message, $comment_id ) {
 	public function comment_moderation_text( $message, $comment_id ) {
 		$this->setup_data( $comment_id );
 
+		if ( empty( $this->comment->comment_type ) ) {
+			$this->comment->comment_type = __( 'comment', 'yoast-comment-hacks' );
```

Comment:
Seems to me now that this sets the comment type to "comment", the checks in `add_author_line()` and `add_content_line()` will use the "Website" and "Excerpt" strings, and the moderation email will have a wrong content:
![screen shot 2017-02-21 at 12 19 07](https://cloud.githubusercontent.com/assets/1682452/23171461/d96ab898-f852-11e6-9300-0ce063feec11.png)